assets/src/console/pages/gateway-accounts/EditCredentialsPage.tsx

```diff
@@ -59,7 +63,13 @@ const handleSubmit = (
   return async (values: FormValues) => {
     try {
       await updateCredentials({
-        variables: { gatewayAccountId: id, input: values }
+        variables: { gatewayAccountId: id, input: values },
+        refetchQueries: [
```

Comment:
if you specify the query name as a string it will automatically use the original variables.

"GetGatewayAccount"

Reply:
oh ok. who knew.